shell/platform/darwin/macos/framework/Headers/FlutterPluginRegistrarMacOS.h

```diff
@@ -36,11 +36,30 @@ FLUTTER_DARWIN_EXPORT
 @property(nonnull, readonly) id<FlutterTextureRegistry> textures;
 
 /**
- * The default view displaying Flutter content.
+ * The view displaying Flutter content.
  *
- * This method may return |nil|, for instance in a headless environment.
+ * Currently Flutter only supports one view, and this is the view.
  *
- * The default view is a special view operated by single-view APIs.
+ * Flutter plans to support multiple views in the future. Although single-view
+ * APIs will eventually be replaced by their multi-view variants, during the
+ * deprecation period, the single-view APIs will coexist with and work with the
+ * multi-view APIs as if the other views don't exist. To achieve this,
+ * all behaviors of "the single view" (which is called "the implicit view") are
+ * preserved, allowing legacy single-view APIs to continue working, while
+ * new-style views created by new ways must be operated by the upcoming
+ * multi-view APIs.
+ *
+ * Plugins written for a single view can keep operating on this view and expect
+ * unchanged behavior for the implicit view. This includes that:
+ *
+ *  - The first view controller attached to the engine will be linked to the
+ *    implicit view.
```

Comment:
This is only true if the implicit view is enabled. I'd consider removing this line, I'm not sure users need to know about this behavior.

Reply:
If the implicit view is not enabled, then this field is null. This field is the implicit view, therefore to explain this field the users need to understand the implicit view.

Reply:
(As far as I understand)

Reply:
Agreed, but the line is incorrect if the implicit view is disabled:

> The first view controller attached to the engine will be linked to the
> implicit view.

If the implicit view is disabled, the first view controller will create a new view which won't be the implicit view.